security-council-rehearsals/step-1/README.md

```diff
@@ -0,0 +1,268 @@
+# Rehearsal 1 - Welcome to SuperchainOps
+
+## Objective
+
+This rehearsal is intended simply to ensure that all the signers feel
+confident running the tooling and performing the validations required
+to execute an onchain action.
+
+Once completed, you can be sure that if the task is executed, the WelcomeToSuperchainOps contract will return `"Welcome to SuperchainOps, <name>"` from its [`welcome()`](https://etherscan.io/address/0x0fb11b4164894912f079de62699f4cc5a2271f0c#readContract#F2) method.
+
+The call that will be executed can be found in the `build` function of the `WelcomeToSuperchainOps` template.
+
+Note that no onchain actions will be taking place during this
+signing. You won’t be submitting a transaction and your address
+doesn’t even need to be funded. These are offchain signatures produced with your wallet. 
+A Facilitator will collect them and use them to execute the transaction that you approved.
+
+Execution can be finalized by anyone once a threshold of signatures
+are collected, so a Facilitator will do the final execution for
+convenience.
+
+## Approving the transaction
+
+### 1. Update repo and move to the appropriate folder for this rehearsal task:
+
+```
+cd superchain-ops
+git pull
```

Comment:
Don't we need to do `just install` anymore?

Reply:
Added: https://github.com/ethereum-optimism/superchain-ops/pull/1121/commits/33b92a835d32cc48b31ef2489553487c76c14720